libs/core/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -160,6 +160,7 @@ namespace hpx { namespace util {
             "pu_step = 1",
             "pu_offset = 0",
             "numa_sensitive = 0",
+            "loopback_network = 0"
```

Comment:
This is missing a comma at the end of the line.

Reply:
Fixed.